de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/provider/RequirementsAttributeLabelProvider.java

```diff
@@ -94,23 +96,33 @@ public String getColumnText(Object object, int columnIndex) {
 
 				return valueSwitch.getValueString(propertyInstance);
 
+			} else if (columnIndex == VERIFICATION_COLUMN) {
+
+				APropertyInstance propertyInstance = ca.getPropertyInstances().get(REQUIREMENT_VERIFICIATION_PROPERTY_NUMBER);

```

Comment:
Vertification_column and trace column is a duplicate just the return statement is different, here you could save some lines of code

Reply:
Well, its always different properties with differnt XXX_PROPERTY_NUMBER. I moved the two lines in between to a new method now.